frontend/app/services/TouchpointBackend.scala

```diff
@@ -107,10 +107,8 @@ object TouchpointBackend {
     )
   }
 
-  val Normal = TouchpointBackend(BackendType.Default)
-  val TestUser = TouchpointBackend(BackendType.Testing)
-
-  val All = Seq(Normal, TestUser)
+  lazy val Normal = TouchpointBackend(BackendType.Default)
```

Comment:
although I should point out that even with this change, if DEV zuora is invalid, you can't use UAT only, as going to mem.thegulocal with a test identity user logged in still tries to access the non UAT zuora for some reason.